README.md

```diff
@@ -63,6 +63,28 @@ This integer specifies the LUKS key size (in bytes).
 ##### `encryption_luks_version`
 This integer specifies the LUKS version to use.
 
+##### `vdo_deduplication`
+This specifies whether or not the Virtual Data Optimizer will be used.
+When set, duplicate data stored on underlying storage volume will be
+deduplicated resulting in more storage capacity.
+Can be used together with `vdo_compression`(optional) and `vdo_logical_size` (strongly recommended).
+Limit one `storage_volume` per `storage_pool`.
+Underlying volume has to be at least 9 GB.
+
+##### `vdo_compression`
+This specifies whether or not the Virtual Data Optimizer will be used.
+When set, data stored on underlying storage volume will be
+compressed resulting in more storage capacity.
+Can be used together with `vdo_deduplication`(optional) and `vdo_logical_size` (strongly recommended).
+Limit one `storage_volume` per `storage_pool`.
+
+##### `vdo_logical_size`
+When Virtual Data Optimizer is used, this specifies size that underlying volume
```

Comment:
I'm not sure if "underlying volume" makes sense here. In LVM stack you'll have pool with `volume_size` and the VDO LV with `vdo_logical_size` so just "volume" would make more sense to me here.

Reply:
Fixed: Removed word "underlying".